src/features/klant/KlantAanmaken.vue

```diff
@@ -57,15 +56,16 @@
           id="telefoonnummer"
           type="text"
           class="utrecht-textbox utrecht-textbox--html-input"
-          required
         />
       </div>
 
       <div class="warning-container" v-if="telefoonnummerWarning[0]">
-        <span class="warning">{{ telefoonnummerWarning[0] }}</span>
+        <div class="warning">{{ telefoonnummerWarning[0] }}</div>
       </div>
     </div>
 
+    <div class="warning" v-if="formWarning">{{ formWarning }}</div>
```

Comment:
er is een message componentje die zou je hier kunnen gebruiken. zie bv het contactverzoekformulier.vue. 
  ```
  <application-message
      v-if="formWarning "
      :message="formWarning "
      messageType="error"
    />
```